test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -66,7 +66,7 @@ public async Task TestBlockedRequestAsync(MockTracerAgent agent, bool enableSecu
             Func<Task<(HttpStatusCode StatusCode, string ResponseText)>> attack = () => SubmitRequest("/Health/?arg=[$slice]");
             var resultRequests = await Task.WhenAll(attack(), attack(), attack(), attack(), attack());
             agent.SpanFilters.Add(s => s.Tags["http.url"].IndexOf("Health", StringComparison.InvariantCultureIgnoreCase) > 0);
-            var spans = agent.WaitForSpans(5);
+            var spans = agent.WaitForSpans(expectedSpans);
```

Comment:
This is actually unrelated to the review, it should help fix the security flaky test, thanks to @kevingosse for the tip!